### PR TITLE
Verify Aadhaar QR Secure code  

### PR DESCRIPTION
**Pull request by vuvoth**
## Motivation

Build verify QR secure Code circuit. 

## Resources
- https://uidai.gov.in/images/resource/User_manulal_QR_Code_15032019.pdf

## Merge Checklist

_Choose all relevant options below by adding an `x` now or at any time before submitting for review_

- [ ] PR title adheres to the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) standard
- [ ] PR has a [change set](https://github.com/privacy-scaling-explorations/anon-aadhaar/CHANGELOG.md)
- [ ] PR has been tagged with a change label(s) (i.e. documentation, feature, bug fix, or chore)
- [ ] PR includes documentation if necessary.

## Additional Context

If this is a relatively large or complex change, provide more details here that will help reviewers

### CODE REVIEW

File: packages/circuits/test/QRVerify.test.ts [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const circom_tester = require('circom_tester/wasm/tester')
+
+import { sha256Pad } from '@zk-email/helpers/dist/shaHash'
+import {
+  Uint8ArrayToCharArray,
+  bufferToHex,
+} from '@zk-email/helpers/dist/binaryFormat'
+import { splitToWords } from 'anon-aadhaar-pcd'
+import { convertBigIntToByteArray, decompressByteArray } from '../src/index'
```
Comment (Meyanis95) [inline packages/circuits/test/QRVerify.test.ts:12]: Unnecessary import as functions already in the in the file.

**Review by Meyanis95 — COMMENTED** [review]
Can you specify the scope of this PR in the PR request?

- Does the compile scripts should be in the scope of this PR?
- As this PR is integrating new circuits under /circuits, I think we can remove circuits from pcd package. Also, does the scripts and the resulting files shouldn't be outside of pcd package, either inside /circuits pkg or inside a new pkg /scripts?
- It would be useful to specify the API of the circuits, especially the outputs 
- As discussed, the circuits should be able to "sign" arbitrary data (signal/message), to prevent on-chain front-running and to enable ERC4337 transaction signing, is it something that we need to include in the scope of this PR or something that we'll add later?

**Comment by vuvoth** [comment]
>    * Does the compile scripts should be in the scope of this PR?
>    * As this PR is integrating new circuits under /circuits, I think we can remove circuits from pcd package. Also, does the scripts and the resulting files shouldn't be outside of pcd package, either inside /circuits pkg or inside a new pkg /scripts?
>    * As discussed, the circuits should be able to "sign" arbitrary data (signal/message), to prevent on-chain front-running and to enable ERC4337 transaction signing, is it something that we need to include in the scope of this PR or something that we'll add later?

I think((rethinking)  better better we should do it in another PRs. This PR for verify circuit only.

>    * It would be useful to specify the API of the circuits, especially the outputs

I don't get it! Could you explained more?

**Comment by vuvoth** [comment]
I updated a circuits, with public input.

**Pull request merged**